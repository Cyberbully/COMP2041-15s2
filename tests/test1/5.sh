#!/bin/sh

# Enter pipeline/shell script below.

egrep -i -v "^([^aeiou]*[aeiou]){10,}[^aeiou]*$" marks.txt
