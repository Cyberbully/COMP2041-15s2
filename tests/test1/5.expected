1236732,MARGIE Paranoid Parrot,69
1790880,LAVINA Tough Kid,47
1135135,RANDEE Courage Wolf,92
1394926,KENNETH Art Student Owl,90
1853521,LISA Forever Alone,27
1124179,KATIA Troll Face,47
1202487,CELIA Internet Husband,26
1712327,LERA Derpina/Derpette,63
1904161,SHARDA Pick Up Line Panda,44
1116023,HELEN Inglip,62
1742485,RAYNA Forever Alone,28
1683655,KRISTY Long Neck Reaction Guy,75
1353845,META Pick Up Line Panda,77
1726462,BLAIR Okay,26
1638105,ARDATH Lol,100
1154838,EVELIN Hipster Cat,54
1453606,JEANNETTE Fap,42
1590937,ELENE Socially Lazy Sloth,73
1565370,DALILA Baby Courage Wolf,83
1630648,ALLIE Milk Rage Face,26
1973426,MINNA Business Cat,44
1320848,TARA Smiling Crying Face,72
1562369,SHELLEY Pick Up Line Panda,64
1804214,MARYJANE Baby Courage Wolf,3
1728319,PIPER Internet Husband,90
1085309,ROSALIE Whatâ€™s All This Racket,30
1361360,LOREAN Milk Rage Face,19
1421968,DORIE Tough Kid,82
1460986,MADELINE Ash Pedreiro,62
1462580,STEPHINE Ash Pedreiro,82
1749704,WILLENE Tough Kid,22
1423952,LUPITA Art Student Owl,28
1086754,CATERINA Smiling Crying Face,36
1771904,CASSONDRA I Lied,80
1338453,JOSEPHINA Business Cat,3
1664750,LILLIE Derpina/Derpette,41
1439367,JUDE Me Gusta,48
1713158,JANEL PPFFFTTCH,29
1758471,NEVA Tough Kid,11
1822244,DESIRE Mother of God,37
1064360,SHELBY Courage Wolf,88
1204411,BILLY Socially Awkward Penguin,35
1701988,ELISE Derpina/Derpette,67
1594974,JANET Happy Cat,20
1005726,JOHANNE Lol,48
1436947,KEITH Lol,66
1932911,MERLENE Not Sure If/Fry,10
1965028,MILLY GTFO,89
1181868,MAN Rage Guy,59
1696758,ALEXANDRA Troll Dad,61
1416327,RAEANN I Lied,44
1611501,ISA Fuck Yea(h),67
1619211,GLADYS Lame Pun Coon,92
1944989,TOBI Not Sure If/Fry,60
1681973,AMMIE Not Sure If/Fry,50
1514420,JANAE Courage Wolf,24
1361754,MABLE PPFFFTTCH,48
1158279,BRIDGET GTFO,39
1926455,DELICIA Rage Guy,55
1349702,REYNALDA Troll Dad,17
1460567,DALLAS Fuck Yea(h),50
1236770,CASSY Troll Dad,19
1937360,ADELLE Courage Wolf,58
1354925,GOLDIE Not Sure If/Fry,58
1728697,KEILA Pick Up Line Panda,32
1945866,MABLE Business Cat,30
1691062,MAREN Cereal Guy,52
1052959,LE Forever Alone,41
1847913,LEDA Inglip,10
1777206,PEARLENE Okay,85
1704073,MARILYN Derpina/Derpette,54
1532276,MARVA Aww Yeah,87
1577233,CELESTE Mother of God,9
1899358,SCOTT Challenge Accepted,6
1089441,GAYLE Milk Rage Face,97
1988051,KALYN Courage Wolf,19
1117466,FELICITA Smiling Crying Face,74
1163374,PAMILA Derpina/Derpette,90
1602038,MICKI Lol,94
1463825,DENNY Bachelor Frog,65
1810676,MELVA Lame Pun Coon,10
1109136,PEARLE GTFO,79
1279602,GWYNETH Pick Up Line Panda,78
1978694,LELIA Baby Courage Wolf,99
1435384,YEE Milk Rage Face,26
1407493,TRINIDAD I Lied,26
1019769,SHELLEY True Story Guy,33
1192511,LINN Courage Wolf,70
1325282,ENDA Mother of God,76
1078048,MAI Bachelorette Frog,0
1513150,ANN Not Sure If/Fry,73
1424698,PANDORA Happy Rage Guy,71
1036725,REA I Lied,10
1387276,JENELLE Fuck Yea(h),81
1362395,ELAINA Lame Pun Coon,28
1855738,CRISTIN Lol,7
1341528,JUANA Happy Rage Guy,84
1126661,KAYE Tough Kid,7
1687655,MARIVEL Happy Rage Guy,32
1492615,CLYDE Ash Pedreiro,37
1837476,ARACELY Socially Lazy Sloth,90
1454060,QUYEN Bachelor Frog,95
1762463,ISIDRA No,63
1644136,DOROTHY Business Cat,3
1035713,CAROLYNN Socially Lazy Sloth,61
1706354,INA Rage Guy,84
1359909,STEFANY Happy Cat,85
1164820,NEIDA Oh God,31
1860504,ELIDA Baby Courage Wolf,92
1151254,MYRL Forever Alone,3
1092370,TAMI PPFFFTTCH,0
1399208,DIA I Lied,16
1889166,CANDELARIA Oh God,67
1232019,LADONNA Lame Pun Coon,17
1739770,FLOY Aww Yeah,98
1419359,DANNIE Derpina/Derpette,0
1838449,COLLEEN True Story Guy,79
1197665,FLORENTINA Poker Face,19
1711091,BUNNY Not Sure If/Fry,52
1829929,LOLITA Smiling Crying Face,71
1587933,MAGALY Me Gusta,95
1227330,FIONA Smiling Crying Face,12
1711219,TORY Challenge Accepted,7
1514465,BILLI Y U NO,95
1658847,CHRISTIANA Ash Pedreiro,10
1400741,CAROLYN Happy Cat,69
1311606,ELSE Aww Yeah,91
1530095,CECIL Poker Face,9
1267802,BENNIE Poker Face,90
1968144,RUTHE Bachelor Frog,48
1438060,DELORSE Rage Guy,55
1143084,BIANCA Troll Dad,39
1578415,CARRY Pedo Bear,77
1188451,LAKISHA Troll Face Girl,9
1272141,AN Troll Dad,13
1151597,LORENE Happy Cat,37
1216902,ANGELIA Tough Kid,8
1730406,TAMISHA Bachelorette Frog,57
1305520,KATIA Me Gusta,22
1715885,MADELYN Long Neck Reaction Guy,91
1911647,ROSEANNA Socially Lazy Sloth,28
1724974,BILLYE Tough Kid,100
1151113,CAROLA Insanity Wolf,86
1083352,WILLENA Troll Dad,23
1023455,MA I Lied,51
1812438,SHENITA Paranoid Parrot,2
1202576,CASSEY Challenge Accepted,94
1256704,WAVA Lame Pun Coon,60
1520906,VERDELL Cereal Guy,42
1435956,FRIEDA Lol,32
1518828,MELIA Poker Face,0
1931688,HORTENSE Pedo Bear,10
1007454,TAMATHA PPFFFTTCH,43
1036957,ASUNCION Not Sure If/Fry,46
1544448,JAME Troll Face,6
1603768,MARCELINE Happy Cat,0
1960297,HEIDI Not Sure If/Fry,90
1426406,MYRIAM Bachelor Frog,40
1806977,LYNELL Okay,93
1300301,JALISA Pick Up Line Panda,56
1268460,MARLEEN Troll Dad,24
1523360,ROMONA Bachelor Frog,69
